tests/resources/config.yaml

```diff
@@ -26,44 +26,13 @@ configs:
         unified_weights: [0.7, 0.3]
   - namespace: "sandbox_numalogic_demo3"
     metric_configs:
-      - metric: "rollout_error_rate"
-        composite_keys: [ "namespace", "name", "hash_id" ]
+      - metric: "namespace_app_rollouts_http_request_error_rate"
+        composite_keys: [ "namespace", "name", "app", "rollouts_pod_template_hash" ]
+        static_threshold: 3
+      - metric: "namespace_app_rollouts_http_request_latency"
+        composite_keys: [ "namespace", "name", "app", "rollouts_pod_template_hash" ]
         static_threshold: 3
-        scrape_interval: 30
```

Comment:
Any reason why we are changing the config file in tests? Some tests used to run on the basis of the config here

Reply:
yes, this is to test the default rollout the config. If you are adding a new config, better to add it to test config and test it out against stream.json